_posts/0000-01-02-fredbom.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Cotes for COVID 
* Item 1
As Franklin D. Roosevelt said:
> **Human kindness has never weakened the stamina or softened the fibre of free people. A nation does not have to be cruel to be tough.**

* Item 2
As Sasha Dicter said:
>*Today, give yourself permission to be outrageously kind, irrationally warm, improbably generous. I promise it will be a blast.*